lib/runtime-core/src/state.rs

```diff
@@ -634,6 +694,7 @@ pub mod x64 {
 
         catch_unsafe_unwind(
             || {
+                ::std::intrinsics::breakpoint();
```

Comment:
Wait-- is this leftovers from earlier debugging or is this intended to `longjmp` to some other code? If so, where?

Reply:
This PR is not finished yet :) yeah this is just for debugging.